### ISSUE TITLE
GUIs causing crash (again o.o) 

### ISSUE THREAD

**Issue by ImInTheICU** [0PandaDEV/Nextron#85]
Randomly happening ( i believe it was caused when using /rank then clicking change prefix )

Paper: `[02:37:29 INFO]: This server is running Paper version 1.21-7-master@d23825a (2024-06-19T01:36:53Z) (Implementing API version 1.21-R0.1-SNAPSHOT)`
Minecraft `1.21`

Crash Log:
```
[02:34:27 ERROR]: Could not pass event InventoryClickEvent to Nextron v1.5.6
java.lang.NoClassDefFoundError: Could not initialize class net.wesjd.anvilgui.AnvilGUI
        at Nextron-1.5.6.jar/net.wesjd.anvilgui.AnvilGUI$Builder.<init>(AnvilGUI.java:400) ~[Nextron-1.5.6.jar:?]
        at Nextron-1.5.6.jar/net.pandadev.nextron.guis.features.RankGUIs.lambda$settings$5(RankGUIs.java:75) ~[Nextron-1.5.6.jar:?]
        at Nextron-1.5.6.jar/dev.triumphteam.gui.guis.GuiListener.onGuiClick(GuiListener.java:102) ~[Nextron-1.5.6.jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor28.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[paper-api-1.21-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:81) ~[paper-api-1.21-R0.1-SNAPSHOT.jar:1.21-7-d23825a]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-api-1.21-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[paper-1.21.jar:1.21-7-d23825a]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[paper-1.21.jar:1.21-7-d23825a]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:628) ~[paper-api-1.21-R0.1-SNAPSHOT.jar:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleContainerClick(ServerGamePacketListenerImpl.java:3122) ~[paper-1.21.jar:1.21-7-d23825a]
        at net.minecraft.network.protocol.game.ServerboundContainerClickPacket.handle(ServerboundContainerClickPacket.java:69) ~[paper-1.21.jar:1.21-7-d23825a]
        at net.minecraft.network.protocol.game.ServerboundContainerClickPacket.handle(ServerboundContainerClickPacket.java:33) ~[paper-1.21.jar:1.21-7-d23825a]
        at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:36) ~[paper-1.21.jar:1.21-7-d23825a]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.21.jar:1.21-7-d23825a]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:151) ~[paper-1.21.jar:1.21-7-d23825a]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[paper-1.21.jar:1.21-7-d23825a]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1478) ~[paper-1.21.jar:1.21-7-d23825a]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:201) ~[paper-1.21.jar:1.21-7-d23825a]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:125) ~[paper-1.21.jar:1.21-7-d23825a]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1456) ~[paper-1.21.jar:1.21-7-d23825a]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1449) ~[paper-1.21.jar:1.21-7-d23825a]
        at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:135) ~[paper-1.21.jar:1.21-7-d23825a]
        at net.minecraft.server.MinecraftServer.managedBlock(MinecraftServer.java:1408) ~[paper-1.21.jar:1.21-7-d23825a]
        at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1415) ~[paper-1.21.jar:1.21-7-d23825a]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1260) ~[paper-1.21.jar:1.21-7-d23825a]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:329) ~[paper-1.21.jar:1.21-7-d23825a]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.ArrayIndexOutOfBoundsException: Index 2 out of bounds for length 2 [in thread "Server thread"]
        at Nextron-1.5.6.jar/net.wesjd.anvilgui.version.VersionMatcher.match(VersionMatcher.java:29) ~[Nextron-1.5.6.jar:?]
        at Nextron-1.5.6.jar/net.wesjd.anvilgui.AnvilGUI.<clinit>(AnvilGUI.java:42) ~[Nextron-1.5.6.jar:?]
        at Nextron-1.5.6.jar/net.wesjd.anvilgui.AnvilGUI$Builder.<init>(AnvilGUI.java:400) ~[Nextron-1.5.6.jar:?]
        at Nextron-1.5.6.jar/net.pandadev.nextron.guis.features.RankGUIs.lambda$manualRankCreation$17(RankGUIs.java:191) ~[Nextron-1.5.6.jar:?]
        ... 26 more
```

**Comment by 0PandaDEV** [comment]
Yes this is caused whenever nextron tries to open a anvil gui for renaming something.

**Comment by 0PandaDEV** [comment]
This issue also has to do with removing the use of anvil guis https://github.com/0PandaDEV/Nextron/issues/81

**Comment by 0PandaDEV** [comment]
fixed with the newest merge into the master from the pr/input-listener branch will be released in 1.5.7 soon
